problem_builder/templates/html/review_step.html

```diff
@@ -57,11 +57,16 @@
 
   <!-- Template for extended feedback: Show extended feedback details when all attempts are used up. -->
   <script type="text/template" id="xblock-step-links-template">
-  <% var q, last_question; %>
+  <% var q, last_question, second_last_question; %>
   <ul class="review-list <%= correctness %>-list">
-      <% for (var question in questions) {{ q = questions[question]; last_question = question == questions.length - 1; %>
-      <li><a href="#" class="step-link" data-step="<%= q.step %>"><%= _.template(gettext("Question {number}"), {number: q.number}, {interpolate: /\{(.+?)\}/g}) %></a></li>
-      <% }} %>
+      <% for (var question in questions) { %>
+        <% q = questions[question]; last_question = question == questions.length - 1; second_last_question = question == questions.length - 2; %>
```

Comment:
@bradenmacdonald The code in this line is quite dense; could you break the assignments up into multiple lines?